tensorflow/core/api_def/base_api/api_def_ArgMax.pbtxt

```diff
@@ -11,5 +11,17 @@ END
   summary: "Returns the index with the largest value across dimensions of a tensor."
   description: <<END
 Note that in case of ties the identity of the return value is not guaranteed.
+
+Usage:
+
+	```python
```

Comment:
I think you have extra spacing here compared to the python docstring (the code block here starts a few columns after `:` whereas in the Python files it starts on the same column as `:`)